webapp/channels/src/components/user_settings/advanced/user_settings_advanced.tsx

```diff
@@ -880,6 +881,11 @@ export default class AdvancedSettingsDisplay extends React.PureComponent<Props,
             );
         }
 
+        let deactivateAccountSectionDivider = null;
+        if (deactivateAccountSection) {
+            deactivateAccountSectionDivider = <div className='divider-light'/>;
```

Comment:
so why do we need this divider?

Reply:
You were right. We didn't need a divider, I have updated the code.